### PR TITLE
Generic information about electronics

### PR DESCRIPTION
**Pull request by tehmaze**
This commit provides a new general section on electronics. The page contains details about recommendations for electronics, such as wire types, color conventions and information on electrical safety.

### CODE REVIEW

File: docs/general/electronics.md [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+| Potential | Symbol                     | Colour                        | Note                               |
+| --------- | -------------------------- | ----------------------------- | ---------------------------------- |
+| Ground    | `GND`                      | ![black wire]                 |                                    |
+| 5V        | `5V`                       | ![red wire]                   | For supply lines                   |
+| Signals   | `S`, `Signal`              | ![white wire]                 | For signals such as limit switches |
+| 12V       | `12V`                      | ![yellow wire]                |                                    |
+| 24V       | `24V`, `VCC`, `VMOT`, etc. | ![yellow wire]                |                                    |
+| 48V       | `48V`, `HT`, `VMOT`, etc.  | ![purple wire] ![orange wire] | Use a color different from 5V/24V  |
+
+## Wiring
```
Comment (HumbleDeer) [inline docs/general/electronics.md]: Duplicate header name

File: docs/general/electronics.md [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+### Recommended wire gauges
+
+This table specifies recommended minimum gauges for nickle-free stranded copper wires; thicker wire can always be used. The rated ampacities are just a rule of thumb. At the rated limits the voltage drop (increases over distance), insulation temperature limit, thickness, thermal conductivity, and air convection and temperature should all be taken into account.
+
+| Application    | AWG     | Metric   | Max current rating |
+| -------------- | ------- | -------- | ------------------ |
+| Mains / AC     | 18 AWG  | 0.75 mm² | 10A @ 120/240VAC   |
+| Hotend heater  | 20 AWG  | 0.50 mm² | 6A @ 12-24VDC      |
+| Stepper motors | 22 AWG  | 0.32 mm² | 4A @ 12-48VDC      |
+| Signal wires   | 24 AWG  | 0.20 mm² | 2A @ 5-24VDC       |
```
Comment (HumbleDeer) [inline docs/general/electronics.md:106]: Should we add a line for other general DC power situations such as LED strips, lighting, fan wire extensions, etc?

**Review by HumbleDeer — CHANGES_REQUESTED** [review]
't is but one small issue, but I can't fix it for you without collaborator edit perms

File: docs/general/electronics.md (hunk 2 above)
Comment (tehmaze) [reply docs/general/electronics.md:106]: No

**Pull request merged**